packages/components/RELEASE.md

```diff
@@ -4,13 +4,40 @@ Whenever there is an update to the components, and these changes have been appro
 
 Please see the instructions in the [CONTRIBUTING](CONTRIBUTING.md) file for more details about how to setup the project and make changes to the code for these packages.
 
-## Bump and release
+## The release process
 
-Follow the instructions for Changesets in the root [README](../../README.md).
+Release PRs, titled 'Version Packages', are created and/or automatically updated on every PR merge by the [changeset GitHub action](https://github.com/changesets/action).
 
-🚨 **DON'T FORGET**:
+In preparation for a release, login to Vercel CLI and create a canonical URL for the upcoming version:
```

Comment:
we should specify that this doesn't apply to patch releases

Reply:
updated